### DISCUSSION TITLE
Sorting dengan beberapa kondisi

### DISCUSSION THREAD

**Discussion by abbymakarim** [teamalgoritma/community#19] [Q&A] [answered]
<img width="1128" alt="Screen Shot 2022-03-18 at 19 11 11" src="https://user-images.githubusercontent.com/62086533/159000855-5e0923fb-0635-46b3-9cfb-e0e72298d4f2.png">
 
saya ingin mensortir berdasarkan total death cases tetapi hanya untuk continent yang tidak NaN, itu bagaimana ya?

**Comment by dwiknrd** [comment, marked as answer]
Untuk melakukan subset berdasarkan yang tidak mengandung missing value (NaN), bapak dapat mensubset dengan menggunakan method `notna` seperti berikut:
```
covidData[covidData['total_deaths'].notna()]
```
